test/export/BUILD

```diff
@@ -0,0 +1,6 @@
+subinclude("//test/build_defs")
+
+plz_e2e_test(
+    name = "export_src_please_test",
+    cmd = "plz export --output plz-out/plzexport //src/core && plz --repo_root=$(plz query reporoot)/plz-out/plzexport build //src/core",
```

Comment:
Can we verify anything about the output here?

Reply:
We're validating that after the export we're able to build it by the fact that we get an exit code 0. I think that's what I care about.

Reply:
oh, d'oh, I clearly did not read this well enough. Very elegant!